theia-swift-docker/Dockerfile

```diff
@@ -0,0 +1,152 @@
+FROM node:8 as extension-builder
+
+RUN npm install -g vsce
+
+RUN git clone --depth 1 https://github.com/apple/sourcekit-lsp 
+
+WORKDIR /sourcekit-lsp/Editors/vscode
+RUN npm install
+RUN npm run postinstall
+RUN vsce package -o ./sourcekit-lsp.vsix
+
+
+
+FROM satishbabariya/swift:swift-DEVELOPMENT-SNAPSHOT-2019-03-17-a as lsp-builder
```

Comment:
This is a very specific version of your Docker image. Could we instead rely on a more generic tag, e.g. `latest`, that would follow your new releases?

Reply:
okay, I'll update it, btw it will be updated to official swift docker after swift 5 is officially released.